service/league_service/league_rater.py

```diff
@@ -18,4 +18,104 @@ def rate(
         outcome: GameOutcome,
         player_rating: Rating,
     ):
-        return current_score
+        if current_score.game_count is None:
+            return LeagueScore(
+                current_score.division_id,
+                current_score.score,
+                1
+            )
+        if current_score.game_count < config.PLACEMENT_GAMES - 1:  # This check is before we increase game_count
+            return LeagueScore(
+                current_score.division_id,
+                current_score.score,
+                current_score.game_count + 1
+            )
+
+        rating = player_rating[0] - 3 * player_rating[1]
+        player_div = league.get_player_division(current_score.division_id)
+        if current_score.division_id is None or player_div is None:
```

Comment:
I'm not sure what the best way is. It should never happen in the first plce, that the player has a division that does not exist in the season. Losing data is potentially bad, but I think repairing by replacing is better, because we at least keep the gamecount and I see no way how that player would get a valid divison otherwise until a new season starts.